### PR TITLE
Feature/fsar 224 article page and sticky sidebar

### PR DESCRIPTION
**Pull request by EmilyHazlehurst**
Please provide a thorough description of your component and its usage with screenshots.

## Checklist before opening a PR

Before opening a PR, please make sure:

- [x] your work is in a branch based on and synced with the develop branch.
- [x] your scss and twig files are imported in the javascript file. Without this, Webpack won't be able to compile them.
- [x] your newComponent is imported in the index.js file.
- [x] your folder hierarchy follows the proper structure.
- [x] your code follows the coding standards and naming conventions defined.
- [x] your code has been tested as AA accessibility compliant to the WCAG 2.1 AA standard.
- [x] your code has been tested in all the browsers listed in the GDS browser's list, including IE11.
- [x] your code has been tested in all the different screensizes defined.

### CODE REVIEW

**Comment by c-ancia** [comment]
@EmilyHazlehurst your PR contains RegionalVariationBlock. Do you mind creating another branch without it please? I want to make sure we only merge what relates to the ticket please. :)

File: src/index.js [hunk 1]
```diff
@@ -70,6 +70,8 @@ import breadcrumbJs from './components/general/Breadcrumb/breadcrumb';
 import pagination from "./components/search/Pagination/pagination";
 import searchBar from "./components/search/SearchBar/searchBar";
 import sortBy from "./components/search/SortBy/sortBy";
+import pdfAndPrintButtons from './components/article/PdfAndPrintButtons/pdfAndPrintButtons';
+import stickySidebar from './components/article/StickySidebar/stickySidebar';
```
Comment (c-ancia) [inline src/index.js:74]: could these 2 be grouped with the other `components/article` elements please?

**Review by c-ancia — CHANGES_REQUESTED** [review]

**Comment by c-ancia** [comment]
@EmilyHazlehurst another "general" request: could you comment your Javascript please? Not sure what all the functions are doing exactly. :) Thanks!

File: src/index.js (hunk 1 above)
Comment (EmilyHazlehurst) [reply src/index.js]: Done

**Comment by EmilyHazlehurst** [comment]
> @EmilyHazlehurst your PR contains RegionalVariationBlock. Do you mind creating another branch without it please? I want to make sure we only merge what relates to the ticket please. :)
> 
> Edit: unless there is a reason why we have these changes?

There's a reason; they are small amends that Rachael asked for so I figured I'd do them here to save time. 

I can split them out into their own PR if you'd like?

File: src/components/article/StickySidebar/stickySidebar.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+      return;
+    }
+
+    // Work out whether the screensize is mobile/desktop
+    function getMode() {
+      return window.innerWidth < 1024 ? 'mobile' : 'desktop';
+    }
+
+    // If on desktop, leave the sidebar in the left region. Otherwise, move it in between the hero and content.
+    function moveSidebar(mode, hero, leftRegion, el) {
```
Comment (c-ancia) [inline src/components/article/StickySidebar/stickySidebar.js:17]: I don't know if we necessarily need to move it, I think we just need to ensure the layout has everything at the right place and in the right order instead.
For instance, the article layout would get (outside of potential hero banner & breadcrumb) a block for the articleHero, a block for the left column & a block for the right column. That way we have the proper layout in mobile, we just need to ensure the articleHero & the right column have the same size/place.

**Comment by c-ancia** [comment]
> > @EmilyHazlehurst your PR contains RegionalVariationBlock. Do you mind creating another branch without it please? I want to make sure we only merge what relates to the ticket please. :)
> > Edit: unless there is a reason why we have these changes?
> 
> There's a reason; they are small amends that Rachael asked for so I figured I'd do them here to save time.
> 
> I can split them out into their own PR if you'd like?

No it's okay, just wanted to be sure we're not committing old or unwanted code. thanks!

File: src/components/article/BackTo/backTo.html.twig [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<div class="back-to">
+  <a href="{{ url }}" class="back-to__link" data-fsa-at="{{ label }}">{{ label }}</a>
```
Comment (c-ancia) [inline src/components/article/BackTo/backTo.html.twig:2]: the data attribute should be lowercased, stripped from any special chars & spaces replaced by hyphens. We could either do it here or make another property for it. Whatever suits you best :)

**Review by c-ancia — CHANGES_REQUESTED** [review]

File: src/components/article/StickySidebar/stickySidebar.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        el.style.width = parentWidth + 'px';
+      }
+    }
+
+    const viewportOffset = el.getBoundingClientRect();
+    const top = viewportOffset.top;
+    const parent = el.parentNode;
+    const hero = document.querySelector('.article-hero');
+    const leftRegion = document.querySelector('.content-layout__left');
+
```
Comment (c-ancia) [inline src/components/article/StickySidebar/stickySidebar.js:34]: It doesn't seem like these variables (hero & leftRegion) are used anymore. Can you remove them please?

**Pull request merged**